client/stylesheets/_responsive.scss

```diff
@@ -325,6 +393,18 @@
     .faqdiv ol, .safetydiv ol, .reportdiv ol {
         margin-left: 10px !important;
     }
+
+    .homeCardLi {
+        width: 140px;
+    }
+
+    .homeCardImg {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `homeCardImg` should be written in lowercase with hyphens